<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistiche</class>
 <widget class="QWidget" name="Statistiche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistiche generali</string>
  </property>
  <widget class="QLabel" name="l_prima">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>651</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Il totale dei clienti tesserati Ã¨:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seleziona l'anno e il mese per vedere l'importo totale:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="line_data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM</string>
   </property>
  </widget>
  <widget class="QListView" name="lista_importo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_terza">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Terza statistica</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_conferma">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>conferma</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_quarta">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quarta statistica</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
